src/kvstore/KVStore.h

```diff
@@ -58,165 +58,376 @@ struct StoreCapability {
 
 class Part;
 /**
- * Interface for all kv-stores
- **/
+ * @brief Interfaces of kvstore
+ */
 class KVStore {
  public:
   virtual ~KVStore() = default;
 
-  // Return bit-OR of StoreCapability values;
+  /**
+   * @brief Return bit-OR of StoreCapability values;
+   *
+   * @return uint32_t Bitwise capability
+   */
   virtual uint32_t capability() const = 0;
 
+  /**
+   * @brief Stop the kvstore
+   */
   virtual void stop() = 0;
 
-  // Retrieve the current leader for the given partition. This
-  // is usually called when E_LEADER_CHANGED error code is returned.
+  /**
+   * @brief Retrieve the current leader for the given partition. This is usually called when
+   * E_LEADER_CHANGED error code is returned.
+   *
+   * @param spaceId
+   * @param partID
+   * @return ErrorOr<nebula::cpp2::ErrorCode, HostAddr> Return HostAddr when succeeded, return
+   * ErrorCode when failed
+   */
   virtual ErrorOr<nebula::cpp2::ErrorCode, HostAddr> partLeader(GraphSpaceID spaceId,
                                                                 PartitionID partID) = 0;
 
+  /**
+   * @brief Return pointer of part manager
+   *
+   * @return PartManager*
+   */
   virtual PartManager* partManager() const {
     return nullptr;
   }
 
-  // Read a single key
+  /**
+   * @brief Read a single key
+   *
+   * @param spaceId
+   * @param partId
+   * @param key
+   * @param value
+   * @param canReadFromFollower Whether check if current kvstore is leader of given partition
```

Comment:
remove "whether".

This param is used in func `checkLeader`. If it is set to true, the current part can serve the read regardless of its role. So essentially, this param just indicates whether the current part can serve the read if the it's not leader, not the same as the current description. I think even just deleting the current description is good enough.

Reply:
👌